testing/cloudi/cloudi.initd

```diff
@@ -0,0 +1,51 @@
+#!/sbin/openrc-run
+
+# With the Erlang VM it is possible to use the -heart erl command line argument
+# (which is possible with cloudi configuration of the /etc/cloudi/vm.args file).
+# To allow the use of the heartbeat monitoring of the Erlang runtime system,
+# it is necessary to rely upon the cloudi script for handling the cloudi pid
+# and not utilize openrc for pidfile creation or signal-based termination.
+# The termination of cloudi utilizes a 65000 millisecond timeout, allowing
+# all services a maximum of 60000 milliseconds for termination with
+# 5000 millisecond for internal Erlang VM delays.  The cloudi script does
+# block for this termination to complete.
+
+# A cloudi installation with root privileges may look like a gaping security
+# hole, due to the possibility of executing "cloudi attach" or
+# "cloudi remote_console" and automatically being given root privileges inside
+# the Erlang shell.  However, it is necessary to be able to allow cloudi to
+# execute any external service OS processes at any time, with any user
+# privileges possible.  The use of cloudi on an isolated system is not a
+# problem and the typical modern use-case of most unix deployments.  So, it is
+# not necessary to utilize a user other than root for executing the cloudi
+# script.
+
+# pidfile path is set during configuration step before installation
+#pidfile=/run/cloudi.pid
+
+# These comments are provided here to encourage comment from the alpine
+# developers that are currently blocking the merge of this port, since there
+# has been no feedback or communication about necessary changes or
+# potential problems.
```

Comment:
Uh, init file is not a diary… Please put this useful yet verbose description to the APKBUILD and leave just brief note here referencing detail description in APKBUILD.